lib/css/CssParser.js

```diff
@@ -829,6 +980,7 @@ class CssParser extends Parser {
 								return processLocalDeclaration(input, start, end);
 							}
 						}
+						if (!currentDeclaration) processDeclaration(input, start, end);
```

Comment:
Wrong

Reply:
You don't need at all extra processing, just use binary search before creation deps